package/MDAnalysis/lib/util.py

```diff
@@ -1274,3 +1275,23 @@ def blocks_of(a, n, m):
                    a.strides[0], a.strides[1])
 
     return np.lib.stride_tricks.as_strided(a, new_shape, new_strides)
+
+
+class Namespace(object):
+    """Class to allow storing attributes in new namespace. """
+    def __getattr__(self, key):
+        # a.this causes a __getattr__ call for key = 'this' 
+        return self.__dict__[key]
+    def __setattr__(self, key, value):
+        # a.this = 10 causes a __setattr__ call for key='this' value=10
+        self.__dict__[key] = value
+    def __delattr__(self, key):
+        del self.__dict__[key]
+    def __eq__(self, other):
+        return self.__dict__ == other.__dict__
+    def __str__(self):
+        return str(self.__dict__)
+    def __len__(self):
+        return len(self.__dict__)
+    def keys(self):
```

Comment:
I have a feeling this was a quick hack I put in when something stopped working (and probably should have commented -_- ); I'll remove it, and follow up if it is used anywhere else.